litmus-portal/frontend/src/pages/ApplicationDashboard/styles.ts

```diff
@@ -96,6 +96,21 @@ const useStyles = makeStyles((theme) => ({
     background: theme.palette.background.paper,
     borderRadius: 4,
     textTransform: 'none',
+    '&:hover': {
+      borderColor: theme.palette.primary.main,
+      boxShadow: `0px 4px 5px -2px ${fade(
+        theme.palette.highlight,
```

Comment:
should the boxshadow be -2px here, May you please review it with figma

Reply:
yes its the same on figma and litmus-ui button too (it's required for transition here) @whitetiger1399